modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/JavaUpgradeFDSDataProviderCheck.java

```diff
@@ -0,0 +1,183 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.source.formatter.check;
+
+import com.liferay.petra.string.StringBundler;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.source.formatter.parser.JavaTerm;
+
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author Michael Cavalcanti
+ */
+public class JavaUpgradeFDSDataProviderCheck extends BaseJavaTermCheck {
+
+	@Override
+	protected String doProcess(
+			String fileName, String absolutePath, JavaTerm javaTerm,
+			String fileContent)
+		throws Exception {
+
+		List<String> importNames = javaTerm.getImportNames();
+
+		String javaTermContent = javaTerm.getContent();
```

Comment:
@michaelmdelcavalcanti, let's rename it to `content` only to keep consistency with the variable `importNames`, I mean, we invoked `getImportNames` and assigned its return to a variable named `importNames`, therefore we should to the same to `getContent`.

Reply:
This is a pattern in others checks, because the content is the content of the javaTerm.

Reply:
Ok, @michaelmdelcavalcanti. Please be aware that Brian may complain about it and if so, please let him know about this existing convention sharing with him code examples. :wink: